src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -108,10 +108,11 @@ public function getItem(string $resourceClass, $id, string $operationName = null
      */
     private function addWhereForIdentifiers(array $identifiers, QueryBuilder $queryBuilder, ClassMetadata $classMetadata)
     {
+        $alias = $queryBuilder->getRootAliases()[0];
         foreach ($identifiers as $identifier => $value) {
             $placeholder = ':id_'.$identifier;
             $expression = $queryBuilder->expr()->eq(
-                'o.'.$identifier,
+                $alias.'.'.$identifier,
```

Comment:
Can be `"$alias.$identifier"` (PHP 7 style)

Reply:
not really a PHP 7 feature, no ?

I personnaly avoid double-quoted string as it may easily contain bugs:

```php
$alias = 'foo';
$identier = 'bar'
echo "$alas.$identifier" // will output "$alas.bar"
```

But it's up to you maintainers to decide

Reply:
Please use:
```
"{$alias}.{$identifier}"
```

Reply:
Done